FrejaEidClient/src/main/java/com/verisec/frejaeid/client/service/OrganisationIdService.java

```diff
@@ -15,55 +15,82 @@
+            throws FrejaEidClientInternalException {
         super(serverAddress, httpService);
         this.pollingTimeoutInMilliseconds = 0;
     }
 
-    public InitiateAddOrganisationIdResponse initiateAdd(InitiateAddOrganisationIdRequest initiateAddOrganisaitonIdRequest) throws FrejaEidClientInternalException, FrejaEidException {
-        return httpService.send(getUrl(serverAddress, MethodUrl.ORGANISATION_ID_INIT_ADD), RequestTemplate.INIT_ADD_ORGANISATION_ID_TEMPLATE, initiateAddOrganisaitonIdRequest, InitiateAddOrganisationIdResponse.class, initiateAddOrganisaitonIdRequest.getRelyingPartyId());
+    public InitiateAddOrganisationIdResponse initiateAdd(
+            InitiateAddOrganisationIdRequest initiateAddOrganisaitonIdRequest)
+            throws FrejaEidClientInternalException, FrejaEidException {
+        return httpService.send(getUrl(serverAddress, MethodUrl.ORGANISATION_ID_INIT_ADD),
+                                RequestTemplate.INIT_ADD_ORGANISATION_ID_TEMPLATE, initiateAddOrganisaitonIdRequest,
+                                InitiateAddOrganisationIdResponse.class,
+                                initiateAddOrganisaitonIdRequest.getRelyingPartyId());
     }
 
-    public OrganisationIdResult getResult(OrganisationIdResultRequest organisationIdResultRequest) throws FrejaEidClientInternalException, FrejaEidException {
-        return httpService.send(getUrl(serverAddress, MethodUrl.ORGANISATION_ID_GET_RESULT), RequestTemplate.ORGANISATION_ID_RESULT_TEMPLATE, organisationIdResultRequest, OrganisationIdResult.class, organisationIdResultRequest.getRelyingPartyId());
+    public OrganisationIdResult getResult(OrganisationIdResultRequest organisationIdResultRequest)
+            throws FrejaEidClientInternalException, FrejaEidException {
+        return httpService.send(getUrl(serverAddress, MethodUrl.ORGANISATION_ID_GET_RESULT),
+                                RequestTemplate.ORGANISATION_ID_RESULT_TEMPLATE, organisationIdResultRequest,
+                                OrganisationIdResult.class, organisationIdResultRequest.getRelyingPartyId());
     }
 
-    public OrganisationIdResult pollForResult(OrganisationIdResultRequest organisationIdResultRequest, int maxWaitingTimeInSec) throws FrejaEidClientInternalException, FrejaEidException, FrejaEidClientPollingException {
+    public OrganisationIdResult pollForResult(OrganisationIdResultRequest organisationIdResultRequest,
+                                              int maxWaitingTimeInSec)
+            throws FrejaEidClientInternalException, FrejaEidException, FrejaEidClientPollingException {
         long pollingEndTime = System.currentTimeMillis() + TimeUnit.SECONDS.toMillis(maxWaitingTimeInSec);
-        while (maxWaitingTimeInSec == 0 || ((System.currentTimeMillis() + pollingTimeoutInMilliseconds) < pollingEndTime)) {
+        while (maxWaitingTimeInSec == 0
+                || ((System.currentTimeMillis() + pollingTimeoutInMilliseconds) < pollingEndTime)) {
```

Comment:
Create new boolean variable for `((System.currentTimeMillis() + pollingTimeoutInMilliseconds) < pollingEndTime)` an then put condition in one line. Same for all pollForResult methods.